src/makeCalculator.js

```diff
@@ -37,7 +37,34 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(num, currentNum) {
```

Comment:
will be better to use more general names for arguments, it's not clear why one of them is `num`, and the second is `currentNum`